### ISSUE TITLE
Skip Content link does not work

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#129]
Screengrab/videos (insert file names here)
[2020 08 18, Skip content link does not work with mouse click.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5098881/2020.08.18.Skip.content.link.does.not.work.with.mouse.click.mp4.zip)

2020 08 18, Skip content link does not work.mp4

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 84

Trusted Tester Test
Skip Repetitive Content

Brief Description
Skip Content link does not work

Steps to reproduce
1)	00:07 – Tab over to the skip content link
2)	00:08 – Click on the skip content link. The only thing that the skip content link will skip is the skip content link, as it will just disappear.

Desired Result
•	The skip content link skips over repetitive content

Actual Result
•	The skip content link disappears

**Comment by subaha-cob** [comment]
Testing at: https://d8-uat.boston.gov/

**Comment by andrebtlima** [comment]
@subaha-cob I'm still not able to click the Skip Content link.

**Comment by subaha-cob** [comment]
@andrebtlima I cannot reproduce that bug. Can you do a video so i can see what it is. Thanks!

**Comment by david-iterators** [comment]
Hi @andrebtlima and @subaha-cob . We have retested this on UAT and this is now working.  Here are the steps followed:

1) Load the page
2) Press the tab key until the Skip Link field is displayed.  Use Shift-Tab to go backwards if you miss the skip link
3) Press the Return key to activate the skip link
4) Continue to press the tab key in order to continue onto the main content. >>>> Note that the repetitive content, such as the side menu should be skipped.
